tests/test_gpu_examples.py

```diff
@@ -4069,3 +4071,78 @@ def test_prefix_tuning_multiple_devices_encoder_decoder_model(self):
         peft_config = PrefixTuningConfig(num_virtual_tokens=10, task_type="SEQ_2_SEQ_LM")
         model = get_peft_model(model, peft_config)
         model.generate(**inputs)  # does not raise
+
+
+@pytest.mark.skipif(not torch.cuda.is_available(), reason="test requires a GPU")
+@pytest.mark.single_gpu_tests
```

Comment:
Is this how we detect if a test is run nightly? If so, fine!

Reply:
GPU tests are skipped for normal CI, and for nightly, they're included. So it's more of an indirect filter.